wp-content/themes/csisnuclearnetwork/assets/_scss/components/_communication-block-widget.scss

```diff
@@ -12,7 +12,7 @@
     @extend %text-style-label-medium;
   }
 
-  h3.daily {
+  h3 .daily {
```

Comment:
This actually breaks the selector, so we need to add a stylelint disable next line.